src/modules/invoices/listeners.ts

```
@@ -10,7 +10,7 @@
  */
 
 import { getLogger } from '@logtape/logtape';
-import { InvoicePaid } from '@/shared/events/definitions';
+import { InvoicePaid, InvoiceRefunded } from '@/shared/events/definitions';
```

ESLint

[error] `@/shared/events/definitions` import should occur after import of `@/shared/database`

(import/order)